### PR TITLE
Detail Info page

### PR DESCRIPTION
**Pull request by camiriveron**
connects #22 

Detail page with repo information.

Unit test for detail info controller.

### CODE REVIEW

File: src/js/dashboard/franklin-repos.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <h3 class="">Franklin Repos</h3>
+    </div>
+    <div class="columns small-6">
+        <a id="dashboard-newrepo" ng-show="!dc.showLoader" href="" title="New Repo" ng-click="dc.getDeployableRepos()">
+            <svg class="dashboard__newrepo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve">
+                <path d="M50.433,0.892c-27.119,0-49.102,21.983-49.102,49.102s21.983,49.103,49.102,49.103s49.101-21.984,49.101-49.103  S77.552,0.892,50.433,0.892z M50.433,93.258c-23.895,0-43.264-19.371-43.264-43.265c0-23.894,19.37-43.264,43.264-43.264  c23.894,0,43.263,19.37,43.263,43.264C93.696,73.887,74.327,93.258,50.433,93.258z" />
+                <path d="M78,50.964C78,47.118,74.882,44,71.036,44H27.964C24.118,44,21,47.118,21,50.964v0.072C21,54.882,24.118,58,27.964,58  h43.072C74.882,58,78,54.882,78,51.036V50.964z" />
+                <path d="M57,28.964C57,25.118,53.882,22,50.036,22h-0.071C46.118,22,43,25.118,43,28.964v43.071C43,75.882,46.118,79,49.964,79  h0.071C53.882,79,57,75.882,57,72.035V28.964z" />
+            </svg>
```
Comment (oudeismetis) [inline src/js/dashboard/franklin-repos.html]: not sure if this is a thing we do on FE, but does it make sense to encapsulate SVGs into their own file and just reference then? Not sure how often they might change.

Comment (recursivefunk) [inline src/js/dashboard/franklin-repos.html]: If you don't anticipate a change, having it inline is probably fine and we do get a slight performance increase as that's one less http request. I'll leave it up to you.

**Comment by recursivefunk** [comment]
:+1:

**Comment by oudeismetis** [comment]
:+1:

**Pull request merged**